src/PythonClient/src/quixstreams/streamconsumer.py

```diff
@@ -149,16 +168,23 @@ def _on_package_received_dispose(self):
 
     @property
     def stream_id(self) -> str:
-        """Get the id of the stream being read"""
+        """
+        Get the ID of the stream being consumed.
 
+        Returns:
+            str: The ID of the stream being consumed.
+        """
         if self._streamId is None:
             self._streamId = self._interop.get_StreamId()
         return self._streamId
 
     @property
     def properties(self) -> StreamPropertiesConsumer:
         """
-        Gets the consumer for accessing timeseries related information of the stream such as parameter definitions and values
+        Gets the consumer for accessing timeseries related information of the stream such as parameter definitions and values.
```

Comment:
this is a bad original copy paste. steram properties aren't related to timeseries, but to stream itself such as the name, metadata, time recorded etc